### DISCUSSION TITLE
Docker image packaging

### DISCUSSION THREAD

**Discussion by B13rg** [deadlock-wiki/deadbot#23] [Ideas]
Would it be useful to create a dockerfile/image for the project?  It might make it easier to run/automate and manage dependencies, but I don't want to create PR/issue noise.  Would you be open to a PR for it?

**Comment by hassayag** [comment]
Yeah I agree, dealing with dependencies is a bit of a pain, so I'd like a better system for running the scripts. 

My automation plan isn't properly set out, but I'm thinking something like:
1. From local machine, upload necessary raw game files to s3 each patch (DONE)
2. Github action that runs on merge to master
3. Downloads latest patch from s3 and runs `main.sh`

In this case, github actions is facilitating that consistent environment to trigger the scripts. That might help with your concerns? 
But in any case, put up a PR and we can go from there

**Comment by B13rg** [comment]
Thank you!  I've gotten started on a PR to dockerize it a bit, with repo secrets it should be not too difficult to pull files from s3

PR: #35
